lightning/src/ln/channelmanager.rs

```diff
@@ -8400,6 +8500,101 @@ mod tests {
 		nodes[1].node.handle_update_fee(&unkown_public_key, &update_fee_msg);
 	}
 
+	#[test]
+	fn test_connection_limiting() {
```

Comment:
I don't think this test is verifying the `unfunded_channel_count` exemptions for outbound channels and 0confs. I'm also not sure the `internal_open_channel` checks are verified to be enforced on outbound connections (`peer_connected()`'s `inbound` flag is always `true`), if it makes sense to test this direction.

Reply:
Yeah good the news outbound always allowed checks. With the small comment remaining, I think we're all fine with test coverage.